pkg/kubectl/cmd/get.go

```diff
@@ -182,7 +184,14 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 
 		// print the current object
 		if !isWatchOnly {
-			if err := printer.PrintObj(obj, out); err != nil {
+			err := printer.PrintObj(obj, out)
```

Comment:
Short if is the recommended style. The variable will be accessible only to that if-else block.